pynamodb/attributes.pyi

```diff
@@ -125,6 +125,8 @@ class TTLAttribute(Attribute[datetime]):
     def __get__(self: _A, instance: None, owner: Any) -> _A: ...
     @overload
     def __get__(self, instance: Any, owner: Any) -> datetime: ...
+    def serialize(self, value: Any) -> Any: ...
```

Comment:
I don't think we need to repeat this, since the signature is the same as in the base class.